Gemfile

```diff
@@ -15,7 +15,7 @@ else
   gem "chef-bin" # rubocop:disable Bundler/DuplicatedGem
 end
 
-gem "cheffish", "~> 17.0"
+gem "cheffish", "~> 17.0.0"
```

Comment:
This will not pull in 17.1.x versions, is that what we want?

Reply:
correct